src/en/temp-release-notes.md

```diff
@@ -585,15 +585,14 @@ when deploying individual charms. The bundle YAML can include a section
 called "bindings", defining the map of endpoint names to space names.
 
 Example bundle.yaml excerpt:
-    ...
+
```

Comment:
I'm happy to do that. I just followed most of the (currently rendered properly) other code boxes on the page. Shall I go through the whole thing?